word2gauss/embeddings.pyx

```diff
@@ -1329,7 +1329,7 @@ cpdef np.ndarray[uint32_t, ndim=2, mode='c'] text_to_pairs(
 
     text is a list of text documents / sentences.
 
-    Each element of the list is a numpy array of uint32_t IDs, with -1
+    Each element of the list is a numpy array of uint32_t IDs, with 4294967295
```

Comment:
Is there a reason why you don't import `UINT32_MAX` directly instead of embedding this value literally?
```python
cdef extern from "stdint.h":
    int UINT32_MAX
```